ports/deltarune/port.json

```diff
@@ -0,0 +1,30 @@
+{
+  "version": 3,
+  "name": "deltarune.zip",
+  "items": [
+    "Deltarune.sh",
+    "deltarune"
+  ],
+  "items_opt": [],
+  "attr": {
+    "title": "Deltarune",
+    "porter": [
+      "Jeod"
+    ],
+    "desc": "In this sequel to Undertale, Kris and Susie are drawn into a mysterious Dark World, where they join forces with Ralsei to close the Dark Fountains and restore balance. Along the way, they encounter strange foes, form unlikely friendships, and uncover new mysteries tied to the world of monsters.",
+    "desc_md": null,
+    "inst": "Download the Windows version from https://tobyfox.itch.io/deltarune and unzip to `ports/deltarune/gamedata`.",
```

Comment:
Is the game always shipped as a zip? Could be nice to add an auto-extractor.

Reply:
Could be, but don't need to.